src/Maker/MakeEntity.php

```diff
@@ -97,26 +97,35 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
     public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
     {
         if ($input->getArgument('name')) {
-            return;
-        }
-
-        if ($input->getOption('regenerate')) {
-            $io->block([
-                'This command will generate any missing methods (e.g. getters & setters) for a class or all classes in a namespace.',
-                'To overwrite any existing methods, re-run this command with the --overwrite flag',
-            ], null, 'fg=yellow');
-            $classOrNamespace = $io->ask('Enter a class or namespace to regenerate', $this->getEntityNamespace(), [Validator::class, 'notBlank']);
+            if (
+                !$input->getOption('broadcast') &&
+                class_exists(Broadcast::class) &&
+                !class_exists($this->generator->createClassNameDetails($entityClassName, 'Entity\\')->getFullName())
+            ) {
+                $description = $command->getDefinition()->getOption('broadcast')->getDescription();
+                $question = new ConfirmationQuestion($description, false);
+                $isBroadcast = $io->askQuestion($question);
+
+                $input->setOption('broadcast', $isBroadcast);
+            }
+            if ($input->getOption('regenerate')) {
+                $io->block([
+                    'This command will generate any missing methods (e.g. getters & setters) for a class or all classes in a namespace.',
+                    'To overwrite any existing methods, re-run this command with the --overwrite flag',
+                ], null, 'fg=yellow');
+                $classOrNamespace = $io->ask('Enter a class or namespace to regenerate', $this->getEntityNamespace(), [Validator::class, 'notBlank']);
 
-            $input->setArgument('name', $classOrNamespace);
+                $input->setArgument('name', $classOrNamespace);
 
-            return;
-        }
+                return;
+            }
 
-        $argument = $command->getDefinition()->getArgument('name');
-        $question = $this->createEntityClassQuestion($argument->getDescription());
-        $entityClassName = $io->askQuestion($question);
+            $argument = $command->getDefinition()->getArgument('name');
+            $question = $this->createEntityClassQuestion($argument->getDescription());
+            $entityClassName = $io->askQuestion($question);
 
-        $input->setArgument('name', $entityClassName);
+            $input->setArgument('name', $entityClassName);
```

Comment:
I'm seeing a failure in the CI: https://github.com/symfony/maker-bundle/pull/976/checks?check_run_id=3670902563#step:14:28

We still nee the old if statement - https://github.com/symfony/maker-bundle/blob/3ca4dcc50af202114324cfad6bbe8775df6fa4b5/src/Maker/MakeEntity.php#L99 - the difference is that, instead of a `return` inside, the `regenerate` and "asking for a class name" - https://github.com/symfony/maker-bundle/blob/3ca4dcc50af202114324cfad6bbe8775df6fa4b5/src/Maker/MakeEntity.php#L103-L119 - should be moved inside if that if statement :)

Reply:
Done ! 👍